tests/datasource/fluent/test_contexts.py

```diff
@@ -311,6 +317,16 @@ def verify_asset_name_cb(request: PreparedRequest) -> CallbackResult:
                     raise ValueError(
                         f"Asset name should not be default - '{DEFAULT_PANDAS_DATA_ASSET_NAME}'"
                     )
+            old_datasource: dict | None = cloud_api_fake_db["datasources"].get(
+                datasource_id
+            )
+            if old_datasource:
+                if (
+                    payload.data.name
+                    != old_datasource["data"]["attributes"]["datasource_config"]["name"]
+                ):
+                    raise NotImplementedError("Unsure how to handle name change")
```

Comment:
What do we mean by this? We don't support changing asset names?

Reply:
Don't support changing ds names